helpers/update-site/formatter.js

```diff
@@ -1,35 +1,44 @@
 const fs = require('fs');
 
+const globby = require('globby');
 const mkdirp = require('mkdirp');
 
 const { copy } = require('./copy');
 const { safeWriteFile } = require('./common');
 const constants = require('./constants');
 
-const FORMATTER_SRC = `${constants.dirs.HINT_PACKAGES}/formatter-html/src`;
+const globbyPattern = 'node_modules/@hint/{,configuration-all/node_modules/@hint/}formatter-html/dist/src';
```

Comment:
Is this because we don't know for sure where `@hint/formatter-html` will end up getting installed by `npm`? What about using `require.resolve('@hint/formatter-html') or `require.resolve('@hint/formatter-html/package.json')` to get us close to the root directory for the HTML formatter?

Reply:
now that you mention that, `@hint/formatter-html` should be always in node_modules (It is added to dependencies) so there is no need to take into account the "configuration" part.

In this case, `globbyPattern` will be: `const globbyPattern = 'node_modules/@hint/formatter-html/dist/src';`

What do you think? do you still prefer to use `require.resolve`?

Reply:
Nah, just referencing the top-level `node_modules` as you suggested is fine.

Reply:
done!